lib/types/index.ts

```diff
@@ -30,6 +30,9 @@ interface TransformerData extends ContractData {
 	};
 	backflowMapping: [BackflowMapping];
 	encryptedSecrets?: any;
+	$transformer?: {
```

Comment:
Don't know if this always exists...

Reply:
You are right. I was just looking to just place it under `encryptedSecrets`, but didn't notice I put it under the wrong one :facepalm: